js/apps/account-ui/test/oid4vci/oid4vci.spec.ts

```diff
@@ -4,7 +4,7 @@ import { login } from "../login";
 const realm = "verifiable-credentials";
 
 test.describe("Verifiable Credentials page", () => {
-  test("Get offer for test-credential.", async ({ page }) => {
+  test.skip("Get offer for test-credential.", async ({ page }) => {
```

Comment:
Yes because the test throws an error, we didn't notice this before this PR because they where silently skipped

I have tried to fix this, but I don't know how to update the realm so that this works, when I update the dropdown becomes empty

Reply:
Got it, then let's merge this PR and log an issue to fix it later.